src/quickpicks/common.ts

```diff
@@ -122,18 +124,30 @@ export async function selectAndroidKeystore (lastUsed) {
 			id: 'last'
 		});
 	}
+	if (savedKeystorePath) {
+		items.push({
+			label: `Saved ${savedKeystorePath}`,
+			id: 'saved'
+		});
+	}
 	const keystoreAction = await quickPick(items, { placeHolder: 'Browse for keystore or use last keystore' });
 	if (keystoreAction.id === 'browse') {
 		const uri = await window.showOpenDialog({ canSelectFolders: false, canSelectMany: false });
 		return uri[0].path;
+	} else if (keystoreAction.id === 'saved') {
+			return savedKeystorePath;
```

Comment:
The path provided could be absolute or relative. Before returning we might need to do something like do for the outputDirectory in packageApp.ts where we check if it's an absolute path and resolve if not. The below does't currently hand a path like `~/Document/mykeystore` though, we can do that in the future
 https://github.com/appcelerator/vscode-appcelerator-titanium/blob/26357e001ac2afc7910c704f9272368dd76c0f78/src/commands/packageApp.ts#L79-L86